cmake/presets/x64-linux-clang.json

```diff
@@ -13,14 +13,18 @@
         "linux",
         "clang"
       ],
-      "displayName": "x64 linux clang static libs"
+      "displayName": "x64 linux clang static libs",
+      "cacheVariables": {
+        "CMAKE_CXX_FLAGS": "-stdlib=libc++"
+      }
     },
     {
       "name": "x64-linux-clang-dynamic",
       "inherits": "x64-linux-clang-static",
       "displayName": "x64 linux clang dynamic libs",
       "cacheVariables": {
-        "BUILD_SHARED_LIBS": true
+        "BUILD_SHARED_LIBS": true,
+        "CMAKE_CXX_FLAGS": "-stdlib=libc++"
       }
```

Comment:
This was needed to fix broken CI builds for clang on Ubuntu. Not strictly related to this PR.